R/process_extract_sds.R

```diff
@@ -36,8 +36,9 @@ process_extract_sds <- function(data = NULL, year, client_lookup = NULL, write_t
 
   outfile <- data %>%
     dplyr::filter(is_date_in_fyyear(
-      convert_year_to_fyyear(substr(.data$sc_latest_submission, 1L, 4L)),
-      .data$record_keydate1, .data$record_keydate2
```

Comment:
This filter returns true for every single entry as it checks whether the record submission date within the start date and end date, which is always true. What we want is to check whether the financial year lays between dates.

Reply:
Great find, your change is correct and it looks like this 'error' also applies to Alarms and Telecare.